extensions/wikia/Rail/templates/RailController_Index.php

```diff
@@ -1,25 +1,27 @@
 <? if ( $isAside ): ?>
 	<aside>
 <? endif; ?>
-	<div id="WikiaRail" class="WikiaRail<?= !empty($isGridLayoutEnabled) ? ' grid-2' : '' ?>">
+	<div id="WikiaRailWrapper" class="WikiaRail<?= !empty($isGridLayoutEnabled) ? ' grid-2' : '' ?>">
```

Comment:
Yes it doesn't. WikiaRailWrapper keeps old WikiaRail styles, WikiaRail for doing floating magic.